pkg/couchdb/ui/.prettierignore

```diff
@@ -0,0 +1,13 @@
+.DS_Store
+node_modules
+/build
+/.svelte-kit
+/package
```

Comment:
You want to add `/dist` here (as that's what we'll be using for built files)

Reply:
You can use this (they can be the same across the packages)

```
.DS_Store
node_modules
.env
.env.*
!.env.example

# Ignore files for PNPM, NPM and YARN
pnpm-lock.yaml
package-lock.json
yarn.lock

# Ignore meta folders
/.svelte-kit
/.rush
/.vscode

# Bundles and built packages
/build
/dist
/package

```

Reply:
☝️ this is `.prettierignore` from my maintenance pr, it is the same for all packages